group_vars/ethiopia_demo/vars.yml

```diff
@@ -4,3 +4,24 @@ app_env: "demo"
 default_country: "ET"
 deploy_env: "ethiopia-demo"
 domain_name: "simple-demo.moh.gov.et"
+ssh_keys:
```

Comment:
❤️

Reply:
Let's create a follow up story to replicate this to the mainline ansible?

Reply:
https://app.shortcut.com/simpledotorg/story/7732/simplify-ssh-key-setup